### DISCUSSION TITLE
Armor stand entity not spawning

### DISCUSSION THREAD

**Discussion by KianBrose** [Minestom/Minestom#2098] [Help] [answered]
I have the following code:
![image](https://github.com/Minestom/Minestom/assets/62078964/6cb88a49-055b-4567-963e-53927eac6d0b)

This works just fine and does create the zombie that rides the player. However, when swapping it to EntityType.ARMOR_STAND it does not spawn anything, why?

**Comment by KianBrose** [comment]
![image](https://github.com/Minestom/Minestom/assets/62078964/bdcf52dc-26e4-451e-9cc7-c32790b0c72c)
It does seem to spawn, but why is it not visible?
![image](https://github.com/Minestom/Minestom/assets/62078964/1e648973-1e6c-49ff-8f27-a7aac4c2eb20)

**Comment by mworzala** [comment]
Cannot repro, waiting on full minimal repro (discussed in discord, just so others dont take time on it)

**Comment by KianBrose** [comment, marked as answer]
Fixed the issue of it being invisible, turns out my texture pack was misformatted and made the texture of the armor stand invisible...
